packages/web-components/src/components/va-radio/va-radio.tsx

```diff
@@ -270,24 +277,29 @@ export class VaRadio {
         {formsHeading}
         <div class="input-wrap">
           <fieldset class="usa-fieldset" aria-labelledby={ariaLabeledByIds}>
-            <legend class={legendClass} part="legend">
+            <legend class={legendClass} part="legend" aria-describedby={messageAriaDescribedbyId} >
               {HeaderLevel ? (
                 <HeaderLevel part="header" aria-describedby={headerAriaDescribedbyId}>{label}</HeaderLevel>
               ) : (
                 label
               )}&nbsp;
               {
                 useFormsPattern === 'multiple' && (
-                  <span id="header-message" class="sr-only">
```

Comment:
These `sr-only`s were still working but I figure it makes sense to future proof them while I'm here